lib/conjur/command/hosts.rb

```diff
@@ -153,7 +159,7 @@ def self.host_layer_roles host
       end
     end
 
-    hosts.desc "[Deprecated] Enroll a new host into conjur"
+    hosts.desc "Enroll a new host into conjur [DEPRECATED]"
```

Comment:
Ok. I think it's worth adding a comment to that effect. Otherwise, it looks like it was forgotten.